dot/parachain/availability-store/messages.go

```diff
@@ -42,7 +41,7 @@ type ErasureChunk struct {
 // QueryChunk query an `ErasureChunk` from the AV store by candidate hash and validator index
 type QueryChunk struct {
 	CandidateHash  parachaintypes.CandidateHash
-	ValidatorIndex uint32
+	ValidatorIndex uint
```

Comment:
why?

Reply:
https://github.com/paritytech/polkadot-sdk/blob/aa68ea58f389c2aa4eefab4bf7bc7b787dd56580/polkadot/node/subsystem-types/src/messages.rs#L505

Reply:
Because this variable is used to call `erasure.ObtainChunks` which accepts a uint.  I've update these to uint32, and casting to uint when calling `erasure.ObtainChunks`